src/js/Component/DataList.js

```diff
@@ -187,6 +198,16 @@ Wui.DataList.prototype = $.extend(new Wui.O(), new Wui.Data(), {
                         maxI = (me.data.length > me.displayMax && me.displayMax > 0) ? me.displayMax : me.data.length,
                         els = [],
                         i = 0;
+
+                    // if Paging is turned on use paging object to get start and end index.
```

Comment:
I like this comment, and I'm excited about this code that can do local sorting even with paging.